dds_web/templates/user/login.html

```diff
@@ -6,50 +6,16 @@
 
 {% block body %}
 
-
-        <p>
-            For information on how to use the Data Delivery System, as well as
-            see a test protocol, go <a
-                href="https://github.com/ScilifelabDataCentre/dds_cli/blob/master/docs/_build/latex/datadeliverysystem.pdf">here</a>.
+<p class="lead">Please use the form below to log in to the SciLifeLab Data Delivery System.</p>
+<div class="row">
+    <div class="col-lg-6">
+        {% include "components/login_form.html" %}
+    </div>
+    <div class="col-lg-6">
+        <p>Typically, you will arrive here because you have recieved an email from a unit that uses
```

Comment:
This text is a bit crap, help writing something better would be appreciated 🙏🏻

Reply:
```
In order to get a DDS account you need an invitation from one of the SciLifeLab units that use the DDS for data delivery. Once you have received the invitation you can register an account and log in to the system.
```
Or something like that?

Reply:
Looks good 👍🏻 Applied..